Library/Format.swift

```diff
@@ -114,14 +117,15 @@ public enum Format {
    - returns: A formatted string.
    */
   public static func date(secondsInUTC seconds: TimeInterval,
-                          dateFormat: String) -> String {
+                          dateFormat: String,
+                          timeZone: TimeZone = AppEnvironment.current.calendar.timeZone) -> String {
```

Comment:
Yeah, that consistency would be nice. A bit of backstory: we use a coalescing optional on the other formatter only because its signature has a dependency on `env` and Swift doesn't allow default arguments to reference other arguments. _E.g._, this doesn't work:

``` swift
func date(
  // ...
  environment env: Environment = AppEnvironment.current,
  timeZone: TimeZone = env.calendar.timeZone
) { // ...
```

Ideally the `TimeZone` is never `nil`, but it's probably clear enough from the signature that on `nil` it falls back to some other default value.